packages/wrangler/src/miniflare-cli/assets.ts

```diff
@@ -335,6 +335,9 @@ async function generateAssetsFetch(
 
 	const generateResponse = (request: MiniflareRequest) => {
 		const url = new URL(request.url);
+		try {
+			url.pathname = decodeURIComponent(url.pathname);
```

Comment:
instead of mutating the url here, I think a better solution is to have something like 
```js
const assetName = decodeURIComponent(url.pathname);
```
and replace references to `url.pathname` later with `assetName` 
<img width="681" alt="image" src="https://user-images.githubusercontent.com/18808/182145839-1ec54d93-45b6-4c85-9bdd-6591f864c298.png">

Reply:
Made these changes